grammars/jsdoc.cson

```diff
@@ -411,6 +430,92 @@
         ]
       }
     ]
+  # Default @param value, highlighted as JavaScript
+  'default-value':
+    'patterns': [
+      # Strings
+      {
+        'begin': '("|\')'
+        'beginCaptures':
+          '0':
+            'name': 'punctuation.definition.string.begin.js'
+        'end': '\\1|(?=\\*/)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.definition.string.end.js'
+        'name': 'string.quoted.js'
+        'patterns': [
+          {
+            'match': '\\\\(?:[^*]|\\*(?!/))'
+            'name': 'constant.character.escape.js'
+          }
+        ]
+      }
+      {
+        # Nested arrays matched to keep brackets balanced
+        'begin': '(\\[)'
+        'beginCaptures':
+          '0':
+            'name': 'source.embedded.js'
+          '1':
+            # NOTE: Should ideally be "punctuation.definition.bracket.square.begin.js"
+            # We're using the old/"incorrect" scope to ensure consistent highlighting.
+            'name': 'meta.brace.square.js'
+        'end': '(\\])|(?=\\*/)'
+        'endCaptures':
+          '0':
+            'name': 'source.embedded.js'
+          '1':
+            'name': 'meta.brace.square.js'
+        'patterns': [
+          {
+            'include': '#default-value'
+          }
+        ]
+      }
+      # Chunk of JS source (excluding nested arrays; see above). Note we "clamp" the
+      # imported highlighting within a fixed range to stop runaway problems with any
+      # begin/end patterns the JavaScript grammar matches.
+      {
+        'match': '(?:[^"\'\\[\\]*]|\\*[^/])+'
+        'name': 'source.embedded.js'
+        'captures':
+          '0':
+            'patterns': [
+              # Unmatched } after a string literal: `"string"}`
```

Comment:
I think you've realized by now that you and I sometimes disagree on what is shippable.  I apologize for the long delays between reviews and comments; that's something I need to work on when something like this occurs.

If you would really like Linguist to include the new JSDoc highlighting, I would encourage you to submit a (hopefully temporary) PR that eliminates the embedded JS highlighting.  Then, we can continue discussing.

I've been trying out different methods to avoid the scope-chopping that's occurring here.  I've seen some success but it'll take more time and I don't want to delay your Linguist release or have it ship with buggy highlighting.